android/.idea/codeStyles/Project.xml

```diff
@@ -1,135 +0,0 @@
-<component name="ProjectCodeStyleConfiguration">
```

Comment:
I think we had this in the repo to share the codeStyle? I am not sure, might need to check when this file was added

Reply:
Hmm yeah it's from this PR: https://github.com/RevenueCat/purchases-hybrid-common/pull/73... I do wonder whether it's needed, considering we now have detekt... But I'm ok bringing it back